File: VB/16_HeaderAndFooter/HeaderAndFooter.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation.Drawing
Imports Spire.Presentation

Namespace HeaderAndFooter
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()

		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()

			presentation.LoadFromFile("..\..\..\..\..\..\Data\HeaderAndFooter.pptx")

			'Add footer
			presentation.SetFooterText("Demo of Spire.Presentation")

			'Set the footer visible
			presentation.FooterVisible = True

			'Set the page number visible
			presentation.SlideNumberVisible = True

			'Set the date visible
			presentation.DateTimeVisible = True

			'Save the document
			presentation.SaveToFile("HeaderAndFooter_result.pptx", FileFormat.Pptx2010)

			'Launch the PPT file
			Process.Start("HeaderAndFooter_result.pptx")
		End Sub
	End Class
End Namespace

File: CS/16_HeaderAndFooter/HeaderAndFooter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation.Drawing;
using Spire.Presentation;

namespace HeaderAndFooter
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {           
            //Create a PPT document
            Presentation presentation = new Presentation();

            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\HeaderAndFooter.pptx");

            //Add footer
            presentation.SetFooterText("Demo of Spire.Presentation");
           
            //Set the footer visible
            presentation.FooterVisible = true;

            //Set the page number visible
            presentation.SlideNumberVisible = true;

            //Set the date visible
            presentation.DateTimeVisible = true;
        
            //Save the document
            presentation.SaveToFile("HeaderAndFooter_result.pptx", FileFormat.Pptx2010);

            //Launch the PPT file
            System.Diagnostics.Process.Start("HeaderAndFooter_result.pptx");
        }
    }
}